style.css

```diff
@@ -9,389 +9,632 @@ body {
   margin: 0;
 }
 
```

Comment:
### Sorry this should be mentioned in earlier stages 👌🏼 

- [ ] Since the project should follow the mobile-first approach which means that all sections should take up the whole available width of the screen that ranges from **min 375px** and **max 768px**, currently, the webpage layout for mobile is just set for the width `377px`, and looks perfectly at that width 👏🏼 but it should behave responsively in the range **min 375px** and **max 768px**.

| Your Design | Figma Design |
|---|---|
| ![image](https://user-images.githubusercontent.com/68030297/230695688-cfe301f5-a471-481a-871b-b2840504f539.png) | ![Screenshot from 2023-04-08 01-37-39](https://user-images.githubusercontent.com/68030297/230695725-5bdf4928-d8cb-4ca4-ac8e-ef59352dff75.png)|